README.md

```diff
@@ -11,3 +11,20 @@ This is a WORK IN PROGRESS repo :)
 # refactoring.nvim
 The Refactoring library based off the Refactoring book by Martin Fowler
 
+## Setup
+**Packer setup:**
+```lua
+use {
+    "ThePrimeagen/refactoring.nvim",
+    require = {
+        {"nvim-lua/plenary.nvim"}
+    }
+}
+```
+**Example Config:**
+```lua
+local refactor = require("refactoring")
+refactor.setup()
+vim.api.nvim_set_keymap("v", "<Leader>re", [[ <Cmd>lua require('refactoring').refactor('Extract Function')<CR>]], {noremap = true, silent = true, expr = false})
```

Comment:
I could include the specific functions to make this easier too!  I just wanted to make friendly string version so that telescope / fzf can be integrated easily.

Reply:
sounds good to me
fyi added a lazyload example